tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install Valkey
+  ansible.builtin.include_tasks: install.yml
+
+- name: Configure Valkey
+  ansible.builtin.include_tasks: configure.yml
+
+- name: Setup Standalone Mode
```

Comment:
this task not required also service file creation task should be part configure.yml